app/models/commConns.js

```diff
@@ -105,6 +105,7 @@ class CommConns {
             }).catch(reject);
           }).catch(reject);
         }
+        return null
```

Comment:
I would not add these manually. My node install was throwing errors and I inserted the null returns because in these situations it's ok that we're not continuing the promise chain. Arguably in the future this could help us track down errors. I mostly added them because the warnings were bugging me.